index.d.ts

```diff
@@ -1387,6 +1440,65 @@ declare module '@amaas/amaas-core-sdk-js' {
         callback?: Function
       ): Promise<IAggregatePNL> | void
     }
+    namespace EOD {
+      function retrieve(
+        {
+          AMId,
+          query
+        }: {
+          AMId: number
+          query: {
+            businessDateStart: string
+            businessDateEnd: string
+            assetIds: string
+          }
+        },
+        callback?: Function
+      ): Promise<IEOD> | void
+    }
+    namespace Curve {
+      function retrieve(
+        { 
+          AMId,
+          resourceId
```

Comment:
Please refactor this so that it is easier to understand. Make the arguments `AMId, businessDate, assetId` so that when this function is called it is easier to understand what should be passed in.